cuebot/src/main/java/com/imageworks/spcue/dao/postgres/LayerDaoJdbc.java

```diff
@@ -490,8 +490,23 @@ public long findPastMaxRSS(JobInterface job, String name) {
 
     @Override
     public void updateTags(JobInterface job, String tags, LayerType type) {
-        getJdbcTemplate().update("UPDATE layer SET str_tags=? WHERE pk_job=? AND str_type=?", tags,
-                job.getJobId(), type.toString());
+        // Split tags by spaces, commas, and pipes to match layer properties behavior
+        String[] tagArray = tags.split("[\\s,|]+");
+        StringBuilder processedTags = new StringBuilder();
+
+        for (String tag : tagArray) {
+            String trimmedTag = tag.trim();
+            if (!trimmedTag.isEmpty()) {
+                if (processedTags.length() > 0) {
+                    processedTags.append(" | ");
+                }
+                processedTags.append(trimmedTag);
+            }
+        }
```

Comment:
Wouldn't this yield `tag1 | tag2 | tag3 |`?

I suggest simplifying this logic and using String.join(" | "):

```java
String tags = String.join(" | ", tagArray);
```